From e81212b6d6ea04bad3eeea9be048d54ee7e0073d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 27 Jan 2024 14:20:27 -0800
Subject: [PATCH 1/3] cdns3-starfive: Include dm/lists.h for
 device_bind_driver_to_node definition

GCC-14 will treat implicit function declarations as errors. Therefore
include the required header to fix

error: implicit declaration of function 'device_bind_driver_to_node' [-Wimplicit-function-declaration]

Upstream-Status: Submitted [https://github.com/starfive-tech/u-boot/pull/56]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 drivers/usb/cdns3/cdns3-starfive.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/cdns3/cdns3-starfive.c b/drivers/usb/cdns3/cdns3-starfive.c
index 631fae21cd..9a48ee6a04 100644
--- a/drivers/usb/cdns3/cdns3-starfive.c
+++ b/drivers/usb/cdns3/cdns3-starfive.c
@@ -11,6 +11,7 @@
 #include <clk.h>
 #include <dm.h>
 #include <dm/device_compat.h>
+#include <dm/lists.h>
 #include <linux/usb/otg.h>
 #include <reset.h>
 #include <regmap.h>
-- 
2.43.0

